WebApplication/Backend/Repositorys/Interfaces/IPatientRepository.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Model.Accounts;
+using Model.Util;
+using System.Collections.Generic;
+
+namespace WebApplication.Backend.Repositorys.Interfaces
+{
+    public interface IPatientRepository
+    {
+        List<Patient> GetAllPatients();
+        Patient GetPatientById(string id);
+        List<Address> GetAddresses(string sqlDml);
```

Comment:
This one should probably be private so it shouldn't be in the interface

Reply:
I removed them from interface. But i can't make them privacy because it is used as a public one. I think these method is inside wrong class patientRepo. Should be addressRepo. By the way we only created interface for it.